src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Cart/Blocks/View/customer_information.html.twig

```diff
@@ -28,45 +28,51 @@
     {{ 'Customer information'|trans({}, 'Admin.Orderscustomers.Feature') }}
   </h3>
   <div class="card-body">
-    <div class="row">
-      {% if cartView.customerInformation.id %}
-        <div class="col">
-          <h2>
-            <a href="{{ path('admin_customers_view', {'customerId': cartView.customerInformation.id }) }}">
-              {% if cartView.customerInformation.gender %}
-                {{ cartView.customerInformation.gender }}
-              {% endif %}
-              {{ cartView.customerInformation.first_name }}
-              {{ cartView.customerInformation.last_name }}
-            </a>
-          </h2>
+    {% if cartView.customerInformation.id %}
+      <p class="mb-0">
+        {{ 'ID:'|trans({}, 'Admin.Orderscustomers.Feature') }}
```

Comment:
```suggestion
        {{ 'Customer ID:'|trans({}, 'Admin.Orderscustomers.Feature') }}
```
Let's use the existing string in the same domain instead of creating a new one. Thank you!